addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1334,6 +1380,14 @@ var SnippetsMenu = Widget.extend({
                         editor.toggleOptions(false);
                     }
                 }
+                // ... if no editors are to be enabled, look if any have been
+                // enabled previously by a click
+                if (!editorToEnable) {
+                     editorToEnable = this.snippetEditors.find(editor => {
+                         return editor.$el && editor.$el.hasClass('o_we_overlay_sticky');
```

Comment:
Just for the sake of not reading internal attribute of a widget: could you make a `isSticky` method in the related class and call it here?